client/src/components/topics.tsx

```diff
@@ -14,13 +14,18 @@ import { State, TopicQuestions } from "types";
 import withLocation from "wrap-with-location";
 import { shouldDisplayPortrait } from "pages";
 
+export interface TopicSytle {
```

Comment:
We should get rid of this named interface and inline it into `Topic` unless it's actually being imported somewhere outside this file?

If we need to keep the interface, then fix the name.

Reply:
We don't need to keep the interface, I passed the interface's attribute as props like in the Chat.tsx